clients/cobol-lsp-vscode-extension/src/test/suite/lsp.spec.test.ts

```diff
@@ -530,15 +532,23 @@ suite("Integration Test Suite", function () {
     const copybookUri = await helper.getUri(copybookPath);
 
     await helper.waitFor(
-      () => vscode.languages.getDiagnostics(progUri).length === 1,
+      () => vscode.languages.getDiagnostics(progUri).length === 2,
     );
     const diagnostics = vscode.languages.getDiagnostics(progUri);
     assert.strictEqual(
       diagnostics.length,
-      1,
+      2,
       "got: " + JSON.stringify(diagnostics),
     );
-    assert.strictEqual(diagnostics[0].message, "Errors inside the copybook");
+    assert.strictEqual(
+      diagnostics[0].message,
+      "Errors inside the copybook",
+      "Test: Show errors only for opened files",
+    );
+    assert.strictEqual(
+      diagnostics[1].message,
+      "Implicit EXIT PROGRAM statement executed",
+    );
```

Comment:
Can the diagnostics order be relied upon?
Similarly, in the tests below, now that multiple messages can be generated.